sci-libs/rdkit/rdkit-2015.03.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
+
+inherit cmake-utils python-r1
+
+DESCRIPTION="Cheminformatics and machine-learning software written in C++ and Python"
+HOMEPAGE="http://www.rdkit.org/"
+SRC_URI="https://github.com/rdkit/rdkit/archive/Release_${PV//./_}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="+python -static-libs"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="dev-libs/boost
+	python? (
+		dev-libs/boost[${PYTHON_USEDEP}]
+		dev-python/numpy[${PYTHON_USEDEP}]
+	)
+	>=dev-db/sqlite-3"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}"/rdkit-Release_${PV//./_}
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_PREFIX:PATH="${EPREFIX}"/usr
+		-DRDK_INSTALL_INTREE=0
+		-DRDK_BUILD_CPP_TESTS=OFF
+	)
+
+	if ! use static-libs; then
+		mycmakeargs+=( -DRDK_INSTALL_STATIC_LIBS=OFF )
+	fi
+
+	if ! use python; then
+		mycmakeargs+=( RDK_BUILD_PYTHON_WRAPPERS= )
```

Comment:
Unfortunately, not for these. The available arguments are [1]:
* `WITH_`
* `ENABLE_`
* `NO_`
* `WANT_`
* `BUILD_`
* `HAVE_`
* `USE_`

Another option is to name the use flag like the option and to use `-DFOO=ON` (cmake-utils_use).
For this use case, I don't see an alternative. Anyway, that is not a problem for 2015.03.1, but 9999 might break. That's one of the reasons why live ebuilds don't contain keywords ;)



[1] https://devmanual.gentoo.org/eclass-reference/cmake-utils.eclass/